README.md

```diff
@@ -1,70 +1,91 @@
-You will also see any lint errors in the console.
+This purpose of this project is to build a simple product that would assist us with our calculations, that is just fully-functional and it does not need to be beautiful and user-friendly (at least in its first version).
 
-### `npm test`
+<div align="center">
+<img src="./calculator.png" alt="screenshot"  width="550" height="480">
+</div>
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+<!-- ABOUT THE PROJECT -->
+## Installation
 
-### `npm run build`
+Download or clone the [repo here](https://github.com/maosan132/calculator.git), enter the directory in the terminal and run the commands:
+* `npm i`
+* `npm run start`
+Your default browser should open the app at the localhost:3000
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### Built With
+This project was built using these technologies.
+* HTML
+* CSS3
+* React
+* Heroku
+* Babel
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+<!-- CONTACT -->
+## Contact
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+👤 **Mauricio Santos**
 
-### `npm run eject`
+- Github: [@maosan132](https://github.com/maosan132)
+- Twitter: [@maosan132](https://twitter.com/maosan132)
+- Linkedin: [maurisantos](https://www.linkedin.com/in/mauricsantos)
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+<!-- ACKNOWLEDGEMENTS -->
+## Acknowledgements
+* [Microverse](https://www.microverse.org/)
+* [The Odin Project](https://www.theodinproject.com/)
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+## 🤝 Contributing
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+Contributions, issues and feature requests are welcome!
 
-## Learn More
+Feel free to check the [issues page](https://github.com/maosan132/rpg-game/issues).
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+1. Fork it (https://github.com/maosan132/rpg-game/fork)
```

Comment:
- [ ] It will be nice if you check this info (URL)

Reply:
Done, thanks.